srcpkgs/sov/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'sov'
+pkgname=sov
+version=0.9b
+revision=1
+archs="x86_64"
```

Comment:
Why is this restricted?

Reply:
On i686 the build fails with
`/usr/include/GL/glew.h:1693:19: error: conflicting types for 'GLintptr'; have 'ptrdiff_t' {aka 'int'}`
On arm archs build fails with
`meson.build:105:3: ERROR: Include dir /usr/include/GLES2 does not exist.`
I didn't go after these issues mainly because I couldn't test on real hardware. But it works on x86_64-musl so I add that.

Reply:
The `meson.build` script is broken and hardcodes paths, this breaks cross compilation, not any specific architecture.

Reply:
True, I check how it is done in other packages and update the PR. The functional tests also fail, have to check that also.